app/retail/emails.py

```diff
@@ -933,12 +933,12 @@ def render_start_work_applicant_expired(interest, bounty):
 def render_new_bounty_roundup(to_email):
     from dashboard.models import Bounty
     from django.conf import settings
-    subject = "Developers, Ads, and Kudos, Oh My!"
-    new_kudos_pks = [2617, 2555, 2623]
+    subject = "April Updates bring May Create."
```

Comment:
> April Updates bring May Create

i dont get it

Reply:
Like April showers bring May flowers?

Reply:
the thing that makes "Like April showers bring May flowers" work is that it rhymes tho :)

Reply:
it kind of rhymes... :)

Reply:
how about April Update brings May Elate?